### ISSUE TITLE
Autoinline in nested types

### ISSUE THREAD

**Issue by havebeenfitz** [krzysztofzablocki/Sourcery#933]
Hello, I'm trying to use a template for public init generation 

```
{% for type in types.structs where type|annotated:"AutoInit" %}
    {% map type.storedVariables into parameters using var %}{{ var.name }}: {{ var.typeName }}{% endmap %}
    {{spacing}}// sourcery:inline:auto:{{ type.name }}.AutoInit
    {{spacing}}    {{ type.accessLevel }} init({{ parameters|join:", " }}) {
    {{spacing}}        {% for variable in type.storedVariables %}
    {{spacing}}        self.{{ variable.name }} = {{ variable.name }}
    {{spacing}}        {% endfor %}
    {{spacing}}    }
    {{spacing}}// sourcery:end
{% endfor %}
```

And in code I have something like this:

```
public final class NewCell: UICollectionViewCell {
  // some code
}

extension NewCell: SomeProtocol {
   // sourcery: AutoInit
    public struct Model {
         let property1: String
         let property2: String
   }
}
```

**What I was expecting after running sourcery:**

```
public final class NewCell: UICollectionViewCell {
  // some code
}

extension NewCell: ConfigurableView {
    // sourcery: AutoInit
    public struct Model {
         let property1: String
         let propery2: String

        // sourcery:inline:auto:NewCell.Model.AutoInit
        public init(property1: String, property2: String) {
            self.property1 = property1
            self.property2 = property2
        }
       // sourcery:end
   }
}
```

**What happens instead:**

```
public final class NewCe
// sourcery:inline:auto:NewCell.Model.AutoInit
        public init(property1: String, property2: String) {
            self.property1 = property1
            self.property2 = property2
        }
// sourcery:end
l: UICollectionViewCell,


extension NewCell: ConfigurableView {
   // sourcery: AutoInit
    public struct Model {
         let property1: String
         let propery2: String
   }
}
```

So it's kind of treating the nested annotated type wrong. Is my template wrong or is it a bug?) I'm using sourcery 1.3.4, latest release

**Comment by Liquidsoul** [comment]
Seems to me like we have a bug with generated code inlining. Could you try previous versions? This way we could find the version that may have introduced the issue.

**Comment by Liquidsoul** [comment]
@havebeenfitz I have performed a test with the template and code you are giving here and this runs fine with the latest Sourcery (`1.3.4` and `master`).
Note that my sample only contains the one source file you are giving with the AutoInit code. Maybe your context is different and you are using the AutoInit template in more files.
Could you try and give more information?

**Comment by havebeenfitz** [comment]
I'm using it in a multi-module workspace, it's the only annotation for one file. I will try to reproduce it in a new simple project as soon as I can, thanks for your help) I will also try some older versions

**Comment by havebeenfitz** [comment]
I was able to reproduce the issue In test project with `1.3.4`. However with `1.2.1` there is no such issue. But in order to get the older version I needed to change run method in build phase to `${SRCROOT}/sourcery/bin/sourcery` as opposed to just `sourcery`.

As I can see from release notes there were major changes:

> Sourcery is now using SwiftSyntax not SourceKit

Maybe it's somehow connected? Or maybe there is something wrong with running through brew

<img width="605" alt="Screenshot 2021-04-05 at 09 39 52" src="https://user-images.githubusercontent.com/31866271/113545469-e154fe00-95f2-11eb-8f79-558dc1d8fff2.png">
